README.md

```diff
@@ -122,3 +122,12 @@ this repository.
 [Snowflake]: https://docs.glaredb.com/docs/data-sources/supported/snowflake.html
 [Postgres]: https://docs.glaredb.com/docs/data-sources/supported/postgres.html
 [Supported data sources]: https://docs.glaredb.com/docs/data-sources/supported/
+
+## Acknowledgements
+
+![https://arrow.apache.org/datafusion/](https://arrow.apache.org/datafusion/_images/DataFusion-Logo-Background-White.png)
+
+
+![https://arrow.apache.org/](https://arrow.apache.org/img/arrow-inverse.png)
```

Comment:
This image is very hard to see at least in "light" mode, which is what I use for GH:

![Acknowledgements](https://github.com/GlareDB/glaredb/assets/11184711/b15834f1-6bc2-481c-918c-faa2d522887d)

I'm kind of wondering if we can mention it in the "About" of the README and use smaller images. One way of doing that would be searching for images that match our needds, or just using a straight up `<image>` html tag as long as the resolution didn't suffer from scaling.

@scsmithr how do you feel about moving to About ?

Reply:
I'm fine with this acknowledgment section, but let's just skip the images for now.